app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: 'users_id'
+  has_many :likes, foreign_key: 'users_id'
+  has_many :posts, foreign_key: 'users_id'
```

Comment:
- [ ] Instead of **user_id** kindly put **author_id** because you are calling it author not user :blush: and it can create an error in your project :thinking:

Reply:
- [ ] **[REQUIRED]** Good job so far :clap:  But as the previous reviewer mentioned , The associations between your different models can be confusing because of user_id and author_id . Please make sure that it looks similar to the one below :arrow_down:  in order t make your application works as expected :+1: 
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```